test/spmd/test_xla_sharding.py

```diff
@@ -638,14 +580,12 @@ def test_2d_tensor_3d_mesh(self):
 
     t1 = ct1.to(xm.xla_device())
     t2 = ct2.to(xm.xla_device())
-    mesh = self._get_mesh((1, self.n_devices, 1))
-    t1 = xs.mark_sharding(t1, mesh, partition_spec=(1, 2))
+    mesh = self._get_mesh((2, self.n_devices // 2, 1))
```

Comment:
wouldn't cpu test just fail at this case? since there is only 1 device.

Reply:
ditto